### ISSUE TITLE
Nit: Unify Mac OS X "About Box" with Java "About Box"

### ISSUE THREAD

**Issue by zapbot** [zaproxy/zaproxy#1496]
```
On the Mac, there is an about box under the standard application name menu.  This is
present on all Mac GUI apps, including ZAP.  When you click "OWASP ZAP --> About OWASP
ZAP", you are presented with a very generic box with the name of the app, logo, version,
and the copyright (which is incorrect in some ZAP Mac builds - fixed in svn already).
 See image.

As the Java help box contains copyrights, license terms, and probably fancier stuff
in a future version, the OS X menu should be modified to load the Java window we already
have.   There is a way to force the Mac to call a different window handler for the
about menu option, some implementation hints can be found in the links below.

http://www.oracle.com/technetwork/articles/javase/javatomac2-138389.html
https://developer.apple.com/library/mac/documentation/Java/Conceptual/Java14Development/07-NativePlatformIntegration/NativePlatformIntegration.html

```

Original issue reported on code.google.com by `veggiespam` on 2015-01-08 17:18:23
<hr>
 * *Attachment: Mac-About-Box.png<br>![Mac-About-Box.png](https://storage.googleapis.com/google-code-attachments/zaproxy/issue-1496/comment-0/Mac-About-Box.png)*

**Comment by thc202** [comment]
The issue report/comments are no longer up-to-date with the code.
For reference @a0d9d62d98b68abbc411c52d8c4d3e82e064c847 and @6657671601c4f450b242de55175d7ff0e8c88e6e
Note, changes were merged to master (trunk) in different commits (while still in SVN).

**Comment by veggiespam** [comment]
There is significant progress with this nit.  

0.  The Mac's ZAP->About menu pulls up the normal ZAP About menu.  There is a second "About OWASP ZAP" menu under help, but that is fine and acceptable.  The original bug reported here is officially fixed.

1.  The ZAP -> Preferences menu now opens up the ZAP setting box.  Awesome.  The keyboard shortcut is cmd-comma (one of a few common Mac ones i've seen).  There is still Tool -> Options with cmd-opt-O being the keyboard shortcut.

2.  File -> Exit still exists.  But so does File -> Exit and delete session.  If the "Exit and Delete" cannot be moved to the Mac menu, then it would be better to leave both of the Exits on the file menu.  

3. As far as I can see, everything is cmd-something.  Thus, this is probably fixed too, but someone should confirm.

Woot.  Progress on a nit.  Thanks to whomever did the fixes.

**Comment by vanderaj** [comment]
I've tested this, and all of the above comments issues seem to be resolved on my mac using the latest code. It is maddening to see "Exit" on the file menu (it would never get through the AppStore process), but I can understand if folks want that left as an extra addition. I love the command-, to get into options. That's the best change

**Comment by kingthorin** [comment]
Based on the feedback can this be closed?

**Comment by veggiespam** [comment]
The original issue, "OWASP ZAP --> About OWASP ZAP" launching a different about-box than "Help --> About OWASP ZAP", is now closed and confirmed in ZAP 2.4.3 Standard.

The stretch-bug-report of "File --> Exit" is still there, but that is a separate discussion (we'd have to fix / move / replicate "Exit and Delete" too into the app-name-menu, etc etc - file a new bug if we care about it.)

As I'm the original filer, please close this bug.
